sdks/v4-sdk/src/PositionManager.test.ts

```diff
@@ -43,12 +45,15 @@ describe('PositionManager', () => {
     []
   )
 
-  const recipient = '0x0000000000000000000000000000000000000003'
-
   const tokenId = 1
   const slippageTolerance = new Percent(1, 100)
   const deadline = 123
 
+  const mockOwner = '0x0000000000000000000000000000000000000001'
```

Comment:
This is the same address as currency0... probably fine but a bit weird? would rather they were different

Reply:
oh yeah especially as the mock owner is signing over currency0 in the tests

Reply:
ah yeah good point